### DISCUSSION TITLE
Запуск тестов в GoLand

### DISCUSSION THREAD

**Discussion by StanislavGrishaev** [ozontech/allure-go#48] [Q&A] [answered]
Привет ! Во-первых, спасибо за проделанную работу, инструмент отличный) 
Есть один момент который не даёт покоя, с начала моменты работы с allure-go - это быстрый запуск тестов по клику в GoLand, а именно по зелёнкой иконке play слева : 

- пример с allure-go - 
![image](https://user-images.githubusercontent.com/115224622/212928297-50709033-3199-4aff-8507-3926624231f8.png)
- пример без allure-go - 
![image](https://user-images.githubusercontent.com/115224622/212928397-720e4b09-c948-4495-8653-71e9b9d194f6.png)

те, при использования фреймворка и при условии что тесты наследуются от allureSuite.Suite - запускать можно только целиком сьют, не каждый тест или кейс по отдельности. 
скажите, я что то упускаю, как то можно запускать в один клик тесты/кейсы и с использованием allure-go ?

**Comment by koodeex** [comment, marked as answer]
Привет! 
Большое спасибо за обращение
Да, к сожалению, на данный момент этот функционал не поддерживается стандартными средствами GoLand
Однако, этот запрос висит в TODO уже какое то время и, наверное, мы сделаем плагин для интеграции, это будет наиболее быстрый и эффективный способ на данный момент
~~Заведите, пожалуйста, Issue в репозиторий~~
Ишью по проблеме - [ISSUE-50](https://github.com/ozontech/allure-go/issues/50)
